From da6792c7b00e4758e33cd86aea8f1c2fa6948148 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Mon, 10 Feb 2020 22:36:42 +0200
Subject: [PATCH 059/105] Revert "tuning statusbar looking"

This reverts commit e38117776802d29955eca83305f5e1c254a91797.

	modified:   packages/SystemUI/res/layout/system_icons.xml
---
 packages/SystemUI/res/layout/system_icons.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/layout/system_icons.xml b/packages/SystemUI/res/layout/system_icons.xml
index 3e638599031..4ee7242729b 100644
--- a/packages/SystemUI/res/layout/system_icons.xml
+++ b/packages/SystemUI/res/layout/system_icons.xml
@@ -54,4 +54,4 @@
             android:visibility="gone"
         />
     </com.android.keyguard.AlphaOptimizedLinearLayout>
-</LinearLayout>
+</LinearLayout>
\ No newline at end of file
-- 
2.17.1

